lib/pure/httpclient.nim

```diff
@@ -1042,6 +1041,9 @@ proc request*(client: HttpClient | AsyncHttpClient, url: Uri | string,
   ## You need to make sure that the ``url`` doesn't contain any newline
   ## characters. Failing to do so will raise ``AssertionDefect``.
   ##
+  ## ``headers`` are HTTP headers that override the ``client.headers`` for
```

Comment:
https://nim-lang.github.io/Nim/contributing.html
> In nim sources, prefer single backticks to double backticks since it's simpler and nim doc supports it (even in rst files with nim rst2html). [...]

Reply:
Ah, alright. I was just following the other docs on this proc. Should I change all of them?

Reply:
just for the code you're touching unless it's a cleanup-only PR